workflow/controller/controller.go

```diff
@@ -1062,6 +1088,15 @@ func (wfc *WorkflowController) newPodInformer(ctx context.Context) cache.SharedI
 	return informer
 }
 
+func (wfc *WorkflowController) newConfigMapInformer() cache.SharedIndexInformer {
+	labelSelector, _ := labels.Parse(indexes.ConfigMapTypeLabel)
+	return v1.NewFilteredConfigMapInformer(wfc.kubeclientset, wfc.GetManagedNamespace(), configMapResyncPeriod, cache.Indexers{
+		indexes.ConfigMapLabelsIndex: indexes.ConfigMapIndexFunc,
+	}, func(opts *metav1.ListOptions) {
+		opts.LabelSelector = labelSelector.String()
```

Comment:
minor - I find that the Goclient code for labels is really hard to use and opaque, I'd like inline the selector as a string